nncf/tensorflow/graph/transformations/commands.py

```diff
@@ -65,31 +68,32 @@ class TFBeforeLayer(TFLayerPoint):
     where the new object should be inserted.
     """
 
-    def __init__(self, layer_name: str, instance_index: int = 0, in_port: int = 0):
+    def __init__(self, layer_name: str, instance_index: int = 0, input_port_id: int = 0):
```

Comment:
What reason of this renaming?

Reply:
For consistency, adjusted this everywhere.